themes/src/main/resources/theme/base/login/messages/messages_en.properties

```
@@ -433,6 +433,8 @@ auth-username-form-display-name=Username
 auth-username-form-help-text=Start sign in by entering your username
 auth-username-password-form-display-name=Username and password
 auth-username-password-form-help-text=Sign in by entering your username and password.
+auth-x509-client-username-form-display-name=X509 Certificate
+auth-x509-client-username-form-help-text=Sign in with a X509 client certificate
```

Should "an" as it is an "Ex-Five-Oh-Nine" certificate, and that starts with a vowel. 

```suggestion
auth-x509-client-username-form-help-text=Sign in with an X509 client certificate
```